src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class Ball {
+    private String color;
+    private int number;
+
+    public String getColor() {
+        return color;
+    }
+
+    public void setColor(String color) {
+        this.color = color;
+    }
+
+    public int getNumber() {
+        return number;
+    }
+
+    public void setNumber(int number) {
+        this.number = number;
+    }
+
+    @Override
+    public String toString() {
+        return "Ball #" + number + ", color - " + color;
```

Comment:
```suggestion
        return "Ball #" + number + ", color - " + color.name();
```

Reply:
color.name(); - it doesn't work

Reply:
Oh, I'm sorry, `color` is `String` already in your class